### DISCUSSION TITLE
Show and tell: an introduction to Retro Debugger and 6502Bench

### DISCUSSION THREAD

**Discussion by c64cryptoboy** [slajerek/RetroDebugger#42] [Show and tell]
I made a 90-minute introduction to reversing with Retro Debugger and 6502Bench that new users may find useful:  https://youtu.be/Tp6G_cRSDwY

**Comment by slajerek** [comment]
Nice video, thanks! 

By the way, I've already fixed that bug with strange characters in debug events comments, it will be in next version to be released 25th of May during Moonshine Dragons demo party.
